keylime/ima.py

```diff
@@ -24,6 +27,8 @@
 START_HASH = (codecs.decode('0000000000000000000000000000000000000000', 'hex'))
 FF_HASH = (codecs.decode('ffffffffffffffffffffffffffffffffffffffff', 'hex'))
 
+ALLOWLIST_CURRENT_VERSION = 1
```

Comment:
nit: Document constant

```suggestion
#: The version of the allowlist that is supported by this keylime release
ALLOWLIST_CURRENT_VERSION = 1
```

Reply:
Done